feedbooks.py

```diff
@@ -115,11 +115,17 @@ def make_link_data(cls, rel, href=None, media_type=None, rights_uri=None,
         links and (at the request of FeedBooks) ignore the other
         formats.
         """
-        if (rel==Hyperlink.GENERIC_OPDS_ACQUISITION
```

Comment:
I can't find the change that made this necessary but I think it's a recent change we discussed regarding treating a link as "mirrored" at its original location rather than downloading it. If we let (e.g.) the MOBI link out of this method, it won't be mirrored, but its original URL will be registered as a delivery mechanism for this license pool.